api/openapi.yaml

```diff
@@ -1250,6 +1252,65 @@ components:
             type: string
           example:
             stripePaymentId: "pi_4OrAkhLvyihio9p51h9iiFnB"
+    LedgerAlreadyExistsProblem:
+      description: Ledger Exists
+      x-go-type-import:
+        path: github.com/openmeterio/openmeter/internal/credit
+      x-go-type: credit.LedgerAlreadyExistsProblemResponse
```

Comment:
Duplication of type required due to this: https://github.com/deepmap/oapi-codegen/pull/1610